### ISSUE TITLE
Idea: ability to pause a running pomodoro

### ISSUE THREAD

**Issue by brainlid** [codito/gnome-pomodoro#171]
Maybe I'm doing it wrong, but I would like to have the ability to **pause** a pomodoro session timer. 

**Imagine Scenario:** 
You are working and about 1/2 way through a pomodoro session. Someone comes over and physically interrupts you. The interruption could be just a moment or it might take you away from your desk for over an hour.

I would like to pause it so I can deal with the interruption then resume my session without starting a new one. Just a thought.

**Comment by kamilprusko** [comment]
You're not doing it wrong. The app just over-simplifies things, it assumes either you are working or having a break. It behaves now more like a break timer.

It would be nice to have. Determining wgether had been interrupted would be error prone or annoying, so having "Interrupt" / "Continue" option in indicator menu I think would be best way to do it.

Should there be another hotkey? I think of adding a hotkey to open the indicator menu once a tasklist is done.

I'm not sure about that, but disabling timer while running and enabling it again, could offer to continue where you left off... May become annoying.

**Comment by brainlid** [comment]
I would vote for a different hotkey for "pause". On my keyboard, I could even map it to the "Pause/Break" key that I never use for anything else. :)

You might want to solicit other's feedback on that though. Thanks for the response.

**Comment by nikolaijlee** [comment]
I would also like this feature, perhaps a pause button on the indicator. I can't seem to get to the long break because I'm always restarting  XD. Thanks for this fine piece of software!

**Comment by Lori2Lori** [comment]
I'd love to use this feature too.

**Comment by michalrus** [comment]
:+1:

**Comment by alexandre-mbm** [comment]
:+1:

**Comment by yuryrodrigues** [comment]
:+1:

**Comment by kamilprusko** [comment]
Done in master branch

![timer-pause](https://cloud.githubusercontent.com/assets/1135423/13379394/a11ee066-de24-11e5-9d57-954eb4373e16.gif)
